.devcontainer/devcontainer.json

```diff
@@ -68,13 +69,13 @@
 	"otherPortsAttributes": {
 		"onAutoForward": "ignore"
 	},
-
 	// Use 'postCreateCommand' to run commands after the container is created.
 	"customizations": {
 		"vscode": {
 			"extensions": [
-				"ms-dotnettools.csdevkit",
+				"ms-dotnettools.vscodeintellicode-csharp",
```

Comment:
Why?

Reply:
To sum up the discussions from the old PR:

- intellicode gives the user a better IntelliSense then without
- intelliCode depends on devkit so it's still their